[closes #6] Improve styling of Contributors section in the README

Vetzio opened the pull request:
This PR is to improve the styling of the Contributors section using HTML. 

The initial commit adjusts the size of the icons, adds a link to user's profiles by clicking their name, and moves the names below their icon.

Vetzio commented:
Need to make the table look better with the 'style' HTML attribute.
Need to investigate why the hyperlinks aren't opening in a new tab even though target="_blank" has been added.
Need to adjust the text alignment to center on the 'tbody'.

@nizarmah Should I not be creating a PR until I've met all of the criteria for the related issue, since I can push commits to the new branch until it's ready?

nizarmah commented:
@Vetzio right now let's care less about "bureaucracy" and more about keeping the momentum going.

So let's target each of these improvements in a single pull request.

Do what feels right for you, and if I have a problem with something I'll let you know and we can discuss it.

Vetzio reviewed (commented):
@nizarmah I'm having some trouble with using the style attribute for the text/image alignment. 

When I add `style="text-align:center"` to the `<tbody>` it seemingly has no effect in GitHub even though it shows up properly in VSCode's preview. I found a handful of posts about this through Google and StackOverflow, but the answers aren't very consistent. 

I've found success in using `align="center"`, but some of the other posts say that the `align` attribute is deprecated and shouldn't be used. Your thoughts?

nizarmah commented:
Hey @Vetzio! That usually means that GitHub has a style that's overwriting the ones you're adding.

Try applying that style to a lower level tag, meaning `tr` or `td` ;)

nizarmah commented:
Hey @Vetzio! Let's close this PR without the centering.

You're applying the `style` in the correct places, but after inspecting, they seem to get removed. 

Thank you though for working on it. Sorry, I wasted your time with it.

Vetzio commented:
Hey @nizarmah! No worries at all. I thought I was going crazy haha, but I did discover some helpful things that I can use moving forward, so the time wasn't wasted. :D 

I've updated this from a draft to ready for review now. 

I also removed the style attributes for now, and I added `align="center"` to the tbody element, as this seems to be a workaround I found in several posts dating between now and several years ago. Let me know if you'd like me to remove it, though!

nizarmah reviewed (approved):
NICE! 

I didn't even know about the `align="center"` trey! Holy shit, touché.

:D that's what I've been wanting to see, let's goooo 🥳
